android/app/src/main/java/com/hathormobile/MainApplication.java

```diff
@@ -65,5 +67,15 @@ public ReactNativeHost getReactNativeHost() {
   public void onCreate() {
     super.onCreate();
     SoLoader.init(this, /* native exopackage */ false);
+
+    try {
```

Comment:
I think we should include the link to the issue in react-native, and also leave a comment about what it is fixing.

Reply:
Done